chromium_src/third_party/blink/renderer/core/frame/local_dom_window.h

```diff
@@ -8,6 +8,8 @@
 
 #include "third_party/abseil-cpp/absl/types/optional.h"
 
+#include "brave/third_party/blink/renderer/core/brave_session_cache.h"
```

Comment:
why does this need brave_session_cache.h?

Reply:
Good catch -- it's no longer needed.